### DISCUSSION TITLE
Welcome to GreenTunnel Discussions!

### DISCUSSION THREAD

**Discussion by SadeghHayeri** [SadeghHayeri/GreenTunnel#86] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by andytouche** [comment]
Great work on the project. Works flawlessly!

Just wanted to know if I can use my local PiHole as the DNS_SERVER in a Docker installation. 

I tried adding the local and Container IP address but got an exec error.

Cheers

**Comment by ibshar** [comment]
Thanks, this works great. Any plans for Android support?

**Comment by yashaskm11** [comment]
Please release the latest docker image for arm64.

**Comment by VladShulg** [comment]
The application for macOS does not work. first it takes a long time to load, then a message appears that the application unexpectedly terminated without opening
